example/flaskr/ingredient.py

```diff
@@ -65,7 +70,6 @@ def getAlternatives(ingredientName):
     return render_template("recipe/ingredient.html", ingredients=alts)
 
 
-@bp.route("/<ingredientName>/alt/<ratingName>")
```

Comment:
Glad to see we're getting rid of these extra pages as I was unsure if people would actually be navigating to them.

Reply:
Yep, this is a good code fix!